plugins/module_utils/network/sonic/config/ospf_area/ospf_area.py

```diff
@@ -974,10 +974,11 @@ def build_area_stub_delete_requests(self, request_root, commands, have):
             # actually clearing stuff means deleting area
             # area having nothing related to stub also ends up in this case. requests will be empty.
             return True, requests, len(requests) > 0
-        elif len(requests) > 0:
-            # clearing some of the settings in stub but not all
-            # commands has to be a subset of have and whatever in it is translated into requests
-            return False, requests, False
+
```

Comment:
Actually, none of the "settings" were getting cleared for the problem case, so this comment isn't completely accurate.

Also, the wording of the existing comment was confusing.

Suggested re-wording for this comment:

Not all stub configuration is being deleted. (It is also possible that none of the stub options requested for deletion match any current configuration. In that case, no stub configuration is being deleted.)